imgui-lwjgl3/src/main/java/imgui/glfw/ImGuiImplGlfw.java

```diff
@@ -281,12 +277,10 @@ private void updateMousePosAndButtons() {
             if (io.getWantSetMousePos()) {
                 glfwSetCursorPos(windowId, mousePosBackup.x, mousePosBackup.y);
             } else {
-                try (MemoryStack stack = stackPush()) {
-                    final DoubleBuffer xPos = stack.mallocDouble(1);
-                    final DoubleBuffer yPos = stack.mallocDouble(1);
-                    glfwGetCursorPos(windowId, xPos, yPos);
-                    io.setMousePos((float) xPos.get(), (float) yPos.get());
-                }
+                final double[] cursorPosX = new double[1];
+                final double[] cursorPosY = new double[1];
```

Comment:
You got it a little bit wrong. The point is to have those variables as class fields. Thus you will instantiate them only once. Otherwise you will load up GC with constantly removing objects you're creating in an app cycle. 

See an example here: https://github.com/SpaiR/imgui-java/blob/master/imgui-lwjgl3/src/test/java/ImGuiGlfwExample.java#L42-L43

Reply:
I see. That's helpful to know, thanks - I'll be making some changes to my own code too!